test_runner/src/main/kotlin/ftl/config/ios/IosFlankConfig.kt

```diff
@@ -25,16 +25,42 @@ data class IosFlankConfig @JsonIgnore constructor(
     @set:JsonProperty("test-targets")
     var testTargets: List<String?>? by data
 
+    @set:CommandLine.Option(
+        names = ["--only-test-configuration"],
+        description = [
+            "Constrains a test action to only test a specified test configuration within a test plan " +
+                    "and exclude all other test configurations. (default: run all test configurations)." +
+                    "Flank can  combine  multiple  constraint options, but -only-test-configuration has precedence over -skip-test-configuration. " +
+                    "Each test configuration name must match the name of a configuration specified in a test plan and is case-sensitive."
+        ]
+    )
+    @set:JsonProperty("only-test-configuration")
+    var onlyTestConfiguration: String? by data
+
+    @set:CommandLine.Option(
+        names = ["--skip-test-configuration"],
+        description = [
+            "Constrains a test action to skip a specified test configuration " +
+                    "and include all other test configurations. (default: run all test configurations). " +
+                    "Flank can combine  multiple constraint options, but -only-test-configuration has precedence over -skip-test-configuration. " +
```

Comment:
IMO those two options are opposite and should be mutually exclusive on validation.